node/node_test.go

```diff
@@ -54,7 +54,9 @@ func testNodeRun(t *testing.T, enableLease bool) {
 	if enableLease {
 		opts = append(opts, WithNodeEnableLeaseV1Beta1(leases, nil))
 	}
-	node, err := NewNodeController(testP, testNode(t), nodes, opts...)
+	testNode := testNode(t)
+	testNodeCopy := testNode.DeepCopy()
```

Comment:
Why is this needed?

Reply:
To ensure we're working on a local copy of the pod.

Reply:
If we use . testNode directly, it's problematic because NewNodeController refers to the pointer, and modifies that object. This causes the race detector to get unhappy.

Reply:
Ah, so you're trying to please the race detector. A comment would have avoided this discussion, so I suggest adding said comment, please.